### DISCUSSION TITLE
Is there a way to easily install a third-party published applet by end users?

### DISCUSSION THREAD

**Discussion by faressoft** [LaunchMenu/LaunchMenu#189] [General] [answered]
I'm working on some useful applets but I'm wondering if there is an easy way to install applets published on GitHub or NPM by end-users who might not be programmers.

**Comment by TarVK** [comment]
Unfortunately, no. There were plans to create this, but development on the project grinded to a halt due to other things going on in life, combined to a lack of interest in LaunchMenu (LM) by others.

The plan was to use NPM to host third party applets on, and use tags to even make an in LM applet browser. But it turned out that there really weren't any good programmatic APIs for npm. Using the cmd line application from within nodejs would be an option, we however would need to figure out how to ship LM in a way that we can access this, without requiring the user to install node/npm themselves. This was also one of the factors that led to us taking a break from working on LM, since an applet installer was the big next thing on the todo-list, but we just didn't make much progress there.

If you have an insights on this front, we would appreciate the help. I personally won't have time to work on LM at the moment, but in several months, I might have some time again. The main problem was/is my lack of knowledge about lower-level concerns related to shipping an application.

**Reply by sancarn** [reply, marked as answer]
If wanted I guess these applets could be shipped with the default application until a mechanism for 3rd party applet installation is doable.
